### PR TITLE
Add Linux/MacOS installation instructions

### PR DESCRIPTION
**Pull request by hankshill**

### CODE REVIEW

File: MacOSLinuxInstall.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+   programs $ git clone https://github.com/uyitroa/osr2mp4-app
+
+   programs $ cd osr2mp4-app
+
+```
+
+6. Now install dependencies for osr2mp4: `pip install -r requirements.txt`.
+
+You should now be able to run osr2mp4. Make sure you are in the osr2mp4 file that you cloned from the repo and run `python osr2mp4`. If this returns `No module XXXX` then you need to run `pip install XXXX` and try again.
```
Comment (uyitroa) [inline MacOSLinuxInstall.md:123]: shouldn't it be `python main.py`?

File: MacOSLinuxInstall.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+          ~ $ cd /home/username/programs
+
+   programs $ git clone https://github.com/uyitroa/osr2mp4-app
+
+   programs $ cd osr2mp4-app
+
+```
+
+6. Now install dependencies for osr2mp4: `pip install -r requirements.txt`.
+
```
Comment (uyitroa) [inline MacOSLinuxInstall.md:122]: I don't remember if python comes with library Pillow by default, if so then the user needs to uninstall PIL and Pillow before installing Pillow-SIMD

Comment (hankshill) [reply MacOSLinuxInstall.md:122]: I didn't have to install them but some people might. I'll add that and fix the other issue this evening.

Comment (uyitroa) [reply MacOSLinuxInstall.md:122]: Yes you don't have to uninstall it, but the app will then use Pillow, which is slower than Pillow-SIMD.

**Pull request merged**